src/canopy/llm/cohere.py

```diff
@@ -0,0 +1,390 @@
+import time
+from copy import deepcopy
+from typing import Union, Iterable, Optional, Any, Dict, List
+
+import cohere
```

Comment:
cohere is an optional dependency, and we import it for any config in `llm/__init__.py` so please follow the try catch we apply in this cases. For example see `src/canopy/tokenizer/llama.py`

Reply:
Please make sure we are not directly import cohere in any place around the project

Reply:
Good catch!! 

@scottmx81 could you please make that change?